src/AsyncBridge/AsyncCompatLibExtensions.cs

```diff
@@ -85,5 +86,342 @@ public static ConfiguredTaskAwaitable ConfigureAwait(this Task task, bool contin
 
         return new ConfiguredTaskAwaitable(task, continueOnCapturedContext);
     }
+  
+    /// <summary>
+    /// Creates a continuation that executes asynchronously when the target Task completes
+    /// </summary>
+    /// <param name="task">The target Task</param>
+    /// <param name="action">The continuation method to execute</param>
+    /// <param name="state">A state object to pass to the continuation</param>
+    /// <returns>A task representing the continuation status</returns>
```

Comment:
What do you think about coping the documentation from https://github.com/Microsoft/referencesource/blob/b31308b03e8bd5bf779fb80fda71f31eb959fe0b/mscorlib/system/threading/Tasks/Task.cs#L3932-L3948?